src/foam/dao/EasyDAO.js

```diff
@@ -1105,42 +1122,16 @@ model from which to test ServiceProvider ID (spid)`,
     {
       name: 'addPropertyIndex',
       type: 'foam.dao.EasyDAO',
-      args: [ { javaType: 'foam.core.PropertyInfo...', name: 'props' } ],
+      args: 'foam.core.PropertyInfo... props',
       code: function addPropertyIndex() {
         this.mdao && this.mdao.addPropertyIndex.apply(this.mdao, arguments);
         return this;
       },
       javaCode: `
-        DAO dao = (DAO) getMdao();
-        if ( dao != null && dao instanceof foam.dao.MDAO ) {
-          ((foam.dao.MDAO) dao).addIndex(props);
-        } else {
-          ((Logger) getX().get("logger")).warning(getName(), "Index not added, no access to MDAO");
-        }
-        return this;
-      `
-    },
-
-    /** Only relevant if cache is true or if daoType
-      was set to MDAO, but harmless otherwise. Adds an existing index
-      to the MDAO.
-      @param index The index to add.
-    */
-    {
-      name: 'addIndex',
```

Comment:
Why is this removed?

Reply:
Deprecated and not used. We replaced them all with addPropertyIndex.